lib/jekyll/tags/highlight.rb

```diff
@@ -67,6 +70,10 @@ def sanitized_opts(opts, is_safe)
 
       OPTIONS_REGEX = %r!(?:\w="[^"]*"|\w=\w|\w)+!
 
+      def cache
+        Jekyll::Cache.new("Jekyll::Tags::HighlightBlock")
```

Comment:
Can't this be inlined into the `:render` method both itself..? Or is it going to be used in another method call in a future PR?

Reply:
Readability and consistency.